src/clic-subcommand-instance.ads

```diff
@@ -42,6 +44,15 @@ package CLIC.Subcommand.Instance is
    --  be replaced by "<Main_Command_Name> <Replacement> <Extra_Args>".
    --  If Alias is already set, it will be silently replaced.
 
+   procedure Load_Aliases (Conf     : CLIC.Config.Instance;
+                           Root_Key : String := "alias");
+   --  Load aliases from the given configuration.
+   --
+   --  All the config keys in the format "<Root_Key>.<sub-key>" are
+   --  processed. If the value of a key is a not null String, <sub-key>
+   --  is registered as an alias for the value (parsed into a list with
+   --  GNAT.OS_Lib.Argument_String_To_List).
```

Comment:
I'm having trouble visualizing this... Maybe having an example here would help?

Reply:
I tried to make it clearer.